taichi/runtime/llvm/runtime.cpp

```diff
@@ -939,8 +997,10 @@ void element_listgen(LLVMRuntime *runtime,
   auto child_get_num_elements = child->get_num_elements;
   auto child_from_parent_element = child->from_parent_element;
 #if ARCH_cuda
+  // Each block processes a parent container
```

Comment:
If my comment above makes sense, then it seems to me that multiple blocks *can* handle the same parent container, just with different sub-ranges of the cells in that container?

Reply:
You are right - I should have used "Each block processes a slice of a parent container" here. Thanks for pointing this out!